File: 2187.py
teste = 1

while True:
    V = int(input())
    if V == 0:
        break

    I = V // 50
    V %= 50

    J = V // 10
    V %= 10

    K = V // 5
    V %= 5

    L = V

    print(f"Teste {teste}")
    print(f"{I} {J} {K} {L}\n")
    teste += 1

File: 2187.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split("\n");

let teste = 1;

for (let line of input) {
    let V = parseInt(line);
    if (V === 0) break;

    let I = Math.floor(V / 50);
    V %= 50;

    let J = Math.floor(V / 10);
    V %= 10;

    let K = Math.floor(V / 5);
    V %= 5;

    let L = V;

    console.log(`Teste ${teste++}`);
    console.log(`${I} ${J} ${K} ${L}\n`);
}
